src/Bridges/lazy_bridge_optimizer.jl

```diff
@@ -45,6 +45,18 @@ function LazyBridgeOptimizer(model::MOI.ModelLike)
         Dict{Tuple{DataType, DataType}, DataType}())
 end
 
+function Base.show(io::IO, B::LazyBridgeOptimizer)
+    s(n) = n == 1 ? "" : "s"
+    indent = " "^get(io, :indent, 0)
+    n_var = length(Variable.bridges(B))
+    n_con = length(Constraint.bridges(B))
+    MOIU._print(io, summary(B))
+    print(io, "\n$(indent)with $(n_var) variable bridge$(s(n_var))")
+    print(io, "\n$(indent)with $(n_con) constraint bridge$(s(n_con))")
```

Comment:
You may be able to create a show for AbstractBridgeOptimizer by creating a function like `print_bridges_summary` that prints nothing for `Constraint.EmptyMap` and prints this line for `Constraint.Map`.

Reply:
Sounds like a great idea :). Let me know what you think of the implementation in the commit I just pushed.